orc8r/tools/packer/magma-test-virtualbox.json

```diff
@@ -54,45 +50,36 @@
           "--cpus",
           "2"
         ]
-      ]
+      ],
+      "name": "magma",
```

Comment:
I was wondering if this should be "magma_test". But then in the docs I could only find the `vm_name` parameter, so maybe we can just remove the `name`.

Reply:
The `name: magma` was already present in the file and I didn't want to risk anything :wink: 
But this could probably be removed.